circle.yml

```diff
@@ -93,11 +93,10 @@ test:
         && ENABLE_KMS_ENCRYPTION=true npm run ft_node
 
     # Run S3 with file Backend ; run ft_tests
-    # TODO remove 'VERSIONING=no' to test versioning when file backend is ready
     - S3BACKEND=file S3VAULT=mem MPU_TESTING=yes npm start
             > $CIRCLE_ARTIFACTS/server_file_awssdk.txt
         & bash wait_for_local_port.bash 8000 40
-        && VERSIONING=no npm run ft_awssdk
+        && npm run ft_awssdk
```

Comment:
the places where this env variable is used in the actual test files should be removed